cmd/server-manager/plugins/events.lua

```diff
@@ -105,10 +106,16 @@ function weatherAPI(raceConfig, serverOpts, apiKey)
             weather["CMWFUseCustomTime"] = 1
             weather["CMWFXTime"] = 0
             weather["CMWFXUseCustomDate"] = 1
+            
+            -- Comment this two lines and uncomment below to prevent night hours
             weather["CMWFXDateUnModified"] = weatherData["dt"]
             weather["CMWFXDate"] = weatherData["dt"] - (3600 * 5 * weather["CMWFXTimeMulti"]) -- don't ask
 
+	        -- Uncomment to force time 5 hour before sunset (18000s) to prevent night
```

Comment:
There are a couple examples of this sort of thing but I'll leave the comment here, could you try to make sure everything is formatted right? Here the next line shouldn't be indented. It depends what program you're using to edit the Lua but lots will have an auto format shortcut, often ctrl + alt + L.

Reply:
I'll look at that (mix of notepad++ and vi)